core/services/helper/main.py

```diff
@@ -154,28 +191,35 @@ def detect_service(port: int) -> ServiceInfo:
 
         for documentation_path in DOCS_CANDIDATE_URLS:
             try:
-                with requests.get(f"http://127.0.0.1:{port}{documentation_path}", timeout=0.2) as response:
+                with httpx.Client(verify=False) as client:
+                    response = client.get(f"http://127.0.0.1:{port}{documentation_path}", timeout=0.2)
                     if response.status_code != http.HTTPStatus.OK:
                         continue
                     info.documentation_url = documentation_path
 
                 # Get main openapi json description file
                 for api_path in API_CANDIDATE_URLS:
-                    with requests.get(f"http://127.0.0.1:{port}{api_path}", timeout=0.2) as response:
+                    with httpx.Client(verify=False) as client:
+                        response = client.get(f"http://127.0.0.1:{port}{api_path}", timeout=0.2)
                         if response.status_code != http.HTTPStatus.OK:
                             continue
                         api = response.json()
                         # Check all available versions
                         ## The ones that provide a swagger-ui
                         for path in api["paths"].keys():
-                            with requests.get(f"http://127.0.0.1:{port}{path}", timeout=0.2) as response:
+                            with httpx.Client(verify=False) as client:
+                                response = client.get(f"http://127.0.0.1:{port}{path}", timeout=0.2)
                                 if "swagger-ui" in response.text:
                                     info.versions += [path]
                 break
 
             except Exception:
                 # This should be avoided by the first try block, but better safe than sorry
                 break
+
+        if info.valid:
+            Helper.FOUND_BLUEOS_SERVICES[port] = info
```

Comment:
We don't want to cache all the services, only those that we know are from BlueOS:
```suggestion
        if info.valid and port in Helper.BLUEOS_PORTS:
            Helper.FOUND_BLUEOS_SERVICES[port] = info
```